test/test_s3_utils.py

```diff
@@ -120,20 +176,35 @@ def test_s3utils_get_keys_for_data():
     assert keys_fs['server'] == keys['server']
 
 
+@pytest.mark.integrated
 def test_s3utils_get_keys_for_staging():
+    # TODO: I'm not sure what this is testing, so it's hard to rewrite
+    #   But I fear this use of env 'staging' implies the GA test environment has overbroad privilege.
+    #   We should make this work without access to 'staging'.
+    #   -kmp 13-Jan-2021
```

Comment:
It has overboard privileges in ways that go well beyond this test - we might consider disabling this test on GA anyway in case it echos the `keys` dictionary, which will contain (temporary, refreshed daily) admin access keys.